### DISCUSSION TITLE
Anyone else think Tauri memory usage is high?

### DISCUSSION THREAD

**Discussion by ogtega** [hoppscotch/hoppscotch#3775] [General]
<img width="618" alt="image" src="https://github.com/hoppscotch/hoppscotch/assets/10700460/fe917f49-7769-4511-b9f9-e36f6c5a0560">

I know there was reluctance to using electron as a desktop wrapper for hoppscotch just from looking at the Issues page of the project.

But I currently have 7 request tabs open and the Tauri process (`tauri://localhost`) is regularly hovering above 1GB of memory usage and runs far worse than most of the electron apps I regularly run.

Is anyone else experiencing this?

**Comment by sureshdsk** [comment]
I've also got similar memory usage. it was less than 10 tabs open.

**Comment by ezzaouia** [comment]
Hi, I'm noticing the same issue with a tauri app.. 
I asked about it in the tauri discord, it seems like there no (easy) way to profile webview..

**Comment by raketa92** [comment]
3 tabs open, 5-8 gb usage. Need to search other options again. Sorry this is because of tauri which used 5-8 gb, but after i close hoppscotch app memory stabilized.
